README.md

```diff
@@ -137,6 +137,12 @@ To build the website go to the project root directory and run:
     $ yarn install # needed only once, or if dependencies change
     $ yarn build   # to perform the build
 
+In case `yarn build` throws the error: **JavaScript heap out of memory**, make the following change within `package.json` 
+and the issue will be resolved: 
+```
+"build": "run-s --max_old_space_size=4096 build:*"
+```
+
```

Comment:
If we need to configure old space for all cases, i.e. the build is broken without this then I would change the `package.json` otherwise if it's an specific issue some contributors might experience but it's not widely and issue for everyone, I'd suggest to set the `NODE_OPTIONS` environment variable:

```suggestion
In case `yarn build` throws the error: **JavaScript heap out of memory**, the issue can be resolved by increasing the memory used by node.js by setting `NODE_OPTIONS` environment variable to include `--max_old_space_size`, for example to increase the old space to 4GB do:

```shell
$ export NODE_OPTIONS="--max_old_space_size=4096"
```

```